contracts/BaseStrategy.sol

```diff
@@ -181,43 +179,16 @@ abstract contract BaseStrategy {
     using SafeMath for uint256;
     using SafeERC20 for IERC20;
     string public metadataURI;
+    uint256 constant MAX_BPS = 10_000;
+    uint256 public changeLimitRatio = 300;
```

Comment:
we need a setter for onlyManagement = (governance or vault.management) modifier to update this in case of manual intervention